tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -125,6 +126,17 @@ def setup_method(self):
         self.log.info("Created IAM user with name %s", self.s3acc_name)
         if not os.path.exists(self.test_dir_path):
             system_utils.make_dirs(self.test_dir_path)
+        self.log.info("Edit deployment file to add sleep(0) to rgw setup command of all data pods.")
+        resp = self.master_node_list[0].get_deployment_name(SERVER_POD_NAME_PREFIX)
+        for each in resp:
+            resp = self.dtm_obj.edit_deployments_for_delay(self.master_node_list[0], each, 'rgw')
+            assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Edit deployment done for all data pods")
+
+        self.log.info("Check the overall status of the cluster.")
```

Comment:
do we want some delay here if already not present? as it takes some time for cluster to get back online.